README.md

```diff
@@ -83,6 +83,15 @@ cmake -DPATH_TO_LLVM=../../../../llvm-project/artifacts/install/ -G "Visual Stud
 
 You can then open `libClangSharp.sln` in Visual Studio, change the configuration to `Release` and build the `ALL_BUILD` project.
 
+If you building on Linux
+```
+git clone https://github.com/microsoft/clangsharp
+mkdir artifacts/bin/native
+cd artifacts/bin/native
+cmake -DPATH_TO_LLVM=/usr/lib/llvm/12/ -G Ninja ../../..
+ninja
+```
```

Comment:
Could you also add the minimal instructions for `makefile`? This otherwise LGTM

Reply:
Everyone should use `cmake --build .` instead :)